### PR TITLE
docs: contributor guidelines

### PR DESCRIPTION
**Pull request by rohitpandey49**
## Project

GSoC Contributor Guidelines

### CODE REVIEW

File: docs/guidelines/index.md [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+
+[Using FOSSology](https://github.com/fossology/fossology/wiki#using-fossology)
+
+[Contribute to FOSSology](https://github.com/fossology/fossology/wiki#get-started-with-development)
+
+[Get Started with Development](https://github.com/fossology/fossology/wiki#get-started-with-development)
+
+[FOSSology Agents](https://github.com/fossology/fossology/wiki/FOSSology-Agents)
+
+There is also a [list of newbie-friendly issues](https://github.com/fossology/fossology/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22).
```
Comment (GMishx) [inline docs/guidelines/index.md:92]: How about using this link: https://github.com/fossology/fossology/contribute ??

File: docs/guidelines/index.md [hunk 2]
```diff
@@ -0,0 +1,176 @@
...
+
+During the coding periods, GSoC contributors are expected to:
+
+- Work on the GSoC project.
+- Follow the [Code Style Best Practices](https://github.com/fossology/fossology/wiki/Coding-Style).
+- Write a short summary of the work done each week, published to FOSSolgy GSoC tracket. It is also advisable to publish in a personal blog as well.
+    - ``It’s okay to say things like <this> and <that> were challenging because of <reason>.``
+- Say something when you are stuck, lost in the code, confused about the objectives, etc.
+- Notify in advance when taking time off and make use of weekends to rest and avoid burnout.
+- Be present online and proactively reach out to the community if required.
```
Comment (GMishx) [inline docs/guidelines/index.md:140]: Please give hints to document weekly progress onto place like: https://fossology.github.io/gsoc/docs/2022/reuse/updates/2022-06-13

File: docs/guidelines/index.md [hunk 3]
```diff
@@ -0,0 +1,176 @@
...
+
+https://medium.com/@shruti-ag/gsoc21-with-fossology-fd97cd2a3924
+
+## Post-GSoC
+
+Congratulations on reaching the end of GSoC! You've accomplished so much, and the journey has only just begun. Once the final results are announced by Google, it's not the end of the road for you. Here are a few options for you to consider:
+
+- Keep the momentum going and continue to develop your project within the FOSSology community.
+- Apply again next year and take on another exciting challenge.
+- Share your knowledge and experience by becoming a mentor in Google Summer of Code for the next year.
```
Comment (GMishx) [inline docs/guidelines/index.md:161]: - Helping us keeping the documentations updated.
- Give presentations/talks about your work during GSoC.

File: static/img/gsoc-fossology.png
Comment (GMishx) [inline static/img/gsoc-fossology.png:1]: Asset already exists at `static/img/gsocHeader.png`

File: static/img/fossology.png
Comment (GMishx) [inline static/img/fossology.png:1]: Can we use the asset `static/img/logo.png` or `static/img/logo.svg` instead?

**Review by GMishx — CHANGES_REQUESTED** [review]

File: docs/guidelines/index.md [hunk 4]
```diff
@@ -0,0 +1,176 @@
...
+
+![Fossology](/img/fossology.png)
+
+**Welcome to the FOSSology GSoC program.**
+
+This page contains information for aspiring interested in participating and helping with the GSoC 2023 program.
+
+
+**Note:**
+> *Fosslogy is participating in [Google Summer Of Code](https://summerofcode.withgoogle.com/programs/2023/organizations/fossology) 2023! You might be interested in our GSoC info pages from previous years: [2022](https://summerofcode.withgoogle.com/programs/2022/organizations/fossology)*
```
[the remark this answers is not in the document]
Comment (rohitpandey49) [reply docs/guidelines/index.md:23]: I understand your concern about the need to update the information on the page every year to reflect FOSSology's participation in GSoC. However, including such information can be helpful for contributors. We can update every year.

File: static/img/fossology.png
Comment (rohitpandey49) [reply static/img/fossology.png:1]: Yes, we can.

File: docs/guidelines/index.md (hunk 1 above)
Comment (rohitpandey49) [reply docs/guidelines/index.md]: Yes sir, we can use.

**Comment by rohitpandey49** [comment]
@GMishx Sir, I have updated the document as per your suggestions. Please check.

**Review by GMishx — APPROVED** [review]

**Pull request merged**